akka-projection-slick/src/main/scala/akka/projection/slick/SlickProjection.scala

```diff
@@ -51,12 +59,14 @@ object SlickProjection {
       databaseConfig: DatabaseConfig[P],
       saveOffsetAfterEnvelopes: Int,
       saveOffsetAfterDuration: FiniteDuration,
+      projectionConfig: ProjectionConfig = ProjectionConfig(),
       handler: SlickHandler[Envelope]): Projection[Envelope] =
     new SlickProjectionImpl(
       projectionId,
       sourceProvider,
       databaseConfig,
       SlickProjectionImpl.AtLeastOnce(saveOffsetAfterEnvelopes, saveOffsetAfterDuration),
+      projectionConfig,
```

Comment:
this is growing, maybe each projection should have its own config object that defines the DB config, the backoff config and the committing strategy

Reply:
Yes, I think that is needed. The committing strategy is something else though, we have https://github.com/akka/akka-projection/issues/59 for that.